docs/en/04-supported-codebases/mmdet.md

```diff
@@ -1,30 +1,175 @@
+
+```{tip}
+When converting mmdet models to tensorrt models, --device should be set to "cuda"
+```
+
+## Model Specification
+
+Before moving on to model inference chapter, let us talk more about the converted model structure which is very important to do model inference.
+
+The converted model locates in the working directory like `mmdeploy_models/mmdet/ort` in the previous example. It includes:
+
+```
+mmdeploy_models/mmdet/ort
+├── deploy.json
+├── detail.json
+├── end2end.onnx
+└── pipeline.json
+```
+
+in which,
+
+- **end2end.onnx**: backend model which can be inferred by ONNX Runtime
+- **deploy.json**: meta information about backend model
+- **pipeline.json**: inference pipeline of mmdeploy SDK
+- **detail.json**: conversion parameters
+
+And the whole package **mmdeploy_models/mmdet/ort** is defined as **mmdeploy SDK model**. In other words, **mmdeploy SDK model** includes not only backend model but also inference meta information.
+
+## Backend model inference
+
+mmdeploy provides a unified API named as `inference_model` to do this job, making all inference backends API transparent to users.
+
+Take the previous converted `end2end.onnx` model as an example,
+
+```python
+from mmdeploy.apis.utils import build_task_processor
+from mmdeploy.utils import get_input_shape, load_config
+import torch
+
+deploy_cfg = 'configs/mmdet/detection/detection_onnxruntime_dynamic.py'
+model_cfg = './faster-rcnn_r50_fpn_1x_coco.py'
+device = 'cpu'
+backend_model = ['./mmdeploy_models/mmdet/ort/end2end.onnx']
+image = './tests/data/tiger.jpeg'
+
+# read deploy_cfg and model_cfg
+deploy_cfg, model_cfg = load_config(deploy_cfg, model_cfg)
+
+# build task and backend model
+task_processor = build_task_processor(model_cfg, deploy_cfg, device)
+model = task_processor.build_backend_model(backend_model)
+
+# process input image
+input_shape = get_input_shape(deploy_cfg)
+model_inputs, _ = task_processor.create_input(image, input_shape)
+
+# do model inference
+with torch.no_grad():
+    result = model.test_step(model_inputs)
+
+# visualize results
+task_processor.visualize(
+    image=image,
+    model=model,
+    result=result[0],
+    window_name='visualize',
+    output_file='output_detection.png')
+```
+
+## SDK model inference
+
+TODO
+
+## Supported models
+
+|                                              Model                                               |         Task         | OnnxRuntime | TensorRT | ncnn | PPLNN | OpenVINO |
+| :----------------------------------------------------------------------------------------------: | :------------------: | :---------: | :------: | :--: | :---: | :------: |
+|            [ATSS](https://github.com/open-mmlab/mmdetection/tree/master/configs/atss)            |   ObjectDetection    |      Y      |    Y     |  N   |   N   |    Y     |
```

Comment:
'ObjectDetection' or 'Object Detection'? Does `ObjectDetection` has specific meanings?

Reply:
no specific meaning, just indicates a task for detecting object